hassio/hassio-navigation.html

```diff
@@ -0,0 +1,95 @@
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../bower_components/app-layout/app-header-layout/app-header-layout.html'>
+<link rel='import' href='../bower_components/app-layout/app-header/app-header.html'>
+<link rel='import' href='../bower_components/app-layout/app-toolbar/app-toolbar.html'>
+<link rel='import' href='../bower_components/paper-icon-button/paper-icon-button.html'>
+<link rel='import' href='../bower_components/paper-tabs/paper-tabs.html'>
+<link rel='import' href='../bower_components/paper-tabs/paper-tab.html'>
+
+<link rel='import' href='../src/util/hass-mixins.html'>
+
+<link rel='import' href='../src/components/ha-menu-button.html'>
+<link rel='import' href='../src/resources/ha-style.html'>
+
+<dom-module id='hassio-navigation'>
+  <template>
+    <style include='iron-flex iron-positioning ha-style'>
+      paper-tabs {
+        margin-left: 12px;
+        --paper-tabs-selection-bar-color: #FFF;
+        text-transform: uppercase;
+      }
+    </style>
+    <app-header-layout has-scrolling-region>
+      <app-header fixed slot='header'>
+        <app-toolbar>            
+          <ha-menu-button narrow='[[narrow]]' show-menu='[[showMenu]]'></ha-menu-button>
+          <paper-icon-button
+            icon='mdi:arrow-left'
+            on-tap='backTapped'
+            hidden='[[!isAddonPage]]'
+          ></paper-icon-button>
+          <div main-title>[[title]]</div>
+        </app-toolbar>
+        <paper-tabs
+          scrollable
+          selected='[[routeData.page]]'
+          attr-for-selected='page-name'
+          on-iron-activate='handlePageSelected'
+          hidden='[[isAddonPage]]'
+        >
+          <paper-tab page-name='dashboard'>Dashboard</paper-tab>
+          <paper-tab page-name='store'>Add-on store</paper-tab>
+          <paper-tab page-name='snapshots'>Snapshots</paper-tab>
+          <paper-tab page-name='advanced'>Advanced</paper-tab>
+        </paper-tabs>
+      </app-header>
+      <slot></slot>
+    </app-header-layout>
+  </template>
+</dom-module>
+
+<script>
+class HassioNavigation extends window.hassMixins.EventsMixin(Polymer.Element) {
+  static get is() { return 'hassio-navigation'; }
+
+  static get properties() {
+    return {
+      page: {
+        type: String,
+        observer: 'pageChanged',
+      },
+      isAddonPage: Boolean,
+      title: String,
+      narrow: Boolean,
+      showMenu: Boolean,
+    };
+  }
+
+  pageChanged(page) {
+    if (page === 'addon') {
+      this.title = 'Hass.io: add-on details';
```

Comment:
I still think that this makes little sense. You show a page to the user but the tab bar is not related to the page they are looking at?

Reply:
I have no tab bar for this view, check the screenshot one comment above yours

Reply:
oooh, I see the hidden attribute now. Sorry.

But then why not split this out? Like, have the component with tab bar be hidden and show a component that shows the details? There is no reason that this component would have to do both?